themes/components-training-theme/src/css/variables/_buttons.scss

```diff
@@ -0,0 +1,60 @@
+$btn-border-radius-lg: 0;
+$btn-border-radius: 0;
+$btn-border-radius-sm: 0;
+
+$btn-primary: () !default;
+$btn-primary: map-deep-merge(
+	(
+		background-color: $primary,
+		border-color: $primary,
+		color: $white,
+		hover: (
+			background-color: $primary-hover,
+			color: $white,
+		),
+		focus: (
+			background-color: $primary-hover,
+			color: $white,
+		),
+		active: (
+			background-color: $primary-active,
+			color: $white,
+		),
+		disabled: (
+			background-color: $primary-disabled,
+		),
+	),
+	$btn-primary
+);
+
+
+$btn-link: () !default;
+$btn-link: map-deep-merge(
+	(	
+		hover: (
+			background-color: $primary-hover,
+			color: $white,
+		),
+		focus: (
+			background-color: $primary-hover,
+			color: $white,
+		),
+		active: (
+			background-color: $primary-active,
+			color: $white,
+		),
+		disabled: (
+			box-shadow: none,
+			text-decoration: none,
+		),
+	),
+	$btn-link
+);
+
+$btn-lg: () !default;
+$btn-lg: map-deep-merge(
+	(
+		width: 13.75rem,
```

Comment:
Is that really necessary?
I think that the button settings are only regarding padding (x and y axis) and font size.